README.md

```diff
@@ -11,16 +11,17 @@ Existing frontend applications include:
 
 ## Installation instructions
 
+1. [Install](https://maven.apache.org/install.html) Maven
 1. [Install](https://docs.docker.com/install/) Docker
 1. Clone this project
 1. Build and Run the provided Solr application
 ```bash
    cd scholars-discovery/solr
    docker build --tag=scholars-solr .
-   docker run -p 9000:9000 scholars-solr
+   docker run -p 8983:8983 scholars-solr
 ```
 1. Build and Run the application
 ```bash
    mvn clean install
-   mvn spring-boot:run -Dspring-boot.run.profiles=dev -Dspring-boot.run.config.location=/app/
+   mvn spring-boot:run -Dspring-boot.run.config.location=src/main/resources/
```

Comment:
the config.location may not be technically necessary if it's just the default application.yml in the classpath